src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -133,12 +136,12 @@ internal Project(Solution solution, ProjectState projectState)
         /// <summary>
         /// The options used when building the compilation for this project.
         /// </summary>
-        public CompilationOptions CompilationOptions => _projectState.CompilationOptions;
+        public CompilationOptions? CompilationOptions => _projectState.CompilationOptions;
 
         /// <summary>
         /// The options used when parsing documents for this project.
         /// </summary>
-        public ParseOptions ParseOptions => _projectState.ParseOptions;
+        public ParseOptions? ParseOptions => _projectState.ParseOptions;
```

Comment:
💭 It would be nice if this was not nullable...

Reply:
As long as we support F#/TypeScript we don't have something else to specify here; "null" is actually the explicit sentinel in that case.